.changes/next-release/Bug Fix-219eee10-03c2-4122-9bf8-30f0c4ce6f88.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Correct code selection range for one-line selections"
```

is this description useful to customers?

please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

at minimum, a prefix that gives a hint about the topic is needed. If Amazon Q is the most relevant topic here, then:

```suggestion
	"description": "Amazon Q: correct code selection range for one-line selections"
```